### PR TITLE
Improved ImportZones

### PR DESCRIPTION
**Pull request by canislobo**
1. Cache now clears when text box is clicked
2. Fixed issue where tool would fail when importing custom zone types
3. Fixed typo

### CODE REVIEW

**Comment by canislobo** [comment]
The third one ("Improved import zone tool") is the only one meant to be merged

File: src/software/js-samples/importZones.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h3>Import devices</h3>
                         <input id="isMetric" type="checkbox" checked="checked" /> <label for="isMetric">Use meters for zone size</label>
                     </p>
                     <p>
-                        <textarea id="content" cols="120" rows="8"></textarea>
+                        <textarea onclick="clear_cache()" id="content" cols="120" rows="8"></textarea>
```
Comment (beefo) [inline src/software/js-samples/importZones.html]: Please use event handler in JS. Similar to `document.addEventListener(`

File: src/software/js-samples/importZones.html [hunk 2]
```diff
@@ -296,6 +296,16 @@ <h2>Help</h2>
                 });
             }
 
+			function clear_cache(){
```
Comment (beefo) [inline src/software/js-samples/importZones.html]: Please use camel case in method names.

File: src/software/js-samples/importZones.html [hunk 3]
```diff
@@ -296,6 +296,16 @@ <h2>Help</h2>
...
+			if(count<1){
```
Comment (beefo) [inline src/software/js-samples/importZones.html]: Formatting. Should have spaces.

File: src/software/js-samples/importZones.html [hunk 4]
```diff
@@ -296,6 +296,16 @@ <h2>Help</h2>
...
+				groupCache = refreshGroupCache(null);
+				//console.log("cleared group cache");
```
Comment (beefo) [inline src/software/js-samples/importZones.html]: Don't include code which has been commented out.

File: src/software/js-samples/importZones.html [hunk 5]
```diff
@@ -296,6 +296,16 @@ <h2>Help</h2>
...
+				zoneTypeCache = refreshZoneTypeCache(null);
+				//console.log("cleared zone cache");
+				count++;
```
Comment (beefo) [inline src/software/js-samples/importZones.html]: What purpose does `count` serve?

**Pull request merged**